GiniVision/Classes/Networking/Extensions/GiniScreenAPICoordinator+Networking.swift

```diff
@@ -132,7 +132,9 @@ extension GiniScreenAPICoordinator: GiniVisionDelegate {
     
     func didCapture(document: GiniVisionDocument) {
         // Analyze document data right away with the Gini SDK for iOS to have results in as early as possible.
-        self.analyzeDocument(visionDocument: document)
+        if document.type != .image { // TODO: add analysis for images but supporting multipage.
```

Comment:
What is this TODO about and why is it not resolved?

Reply:
Basically now the analysis only works for the first page, and the backend it is not ready to test multipages.
I add the TODO so I won't forget about this.

Reply:
I wouldn't understand it, if I saw it in the code in 2 months. I'd write "TODO: Add multipage support".